portfolio/src/main/webapp/canvas.js

```diff
@@ -0,0 +1,42 @@
+var canvas = document.querySelector('canvas');
+canvas.width = window.innerWidth;
+canvas.height = window.innerHeight;
+// c.fillRect(100,100,100,100);
+// c.beginPath();
+// c.moveTo(50,300);
+// c.lineTo(300,100);
+// c.stroke();
+// for (var i = 0; i < 1; i++){
+//     var x = Math.random() * Window.innerWidth;
+//     var y = Math.random() * Window.innerHeight;
+//     c.beginPath();
+//     c.arc(x,y,30,0,Math.PI*2,false);
+//     c.strokeStyle('blue');
+//     c.stroke();
+//}
```

Comment:
Comments are good in code when they're used to describe the code itself (especially if it's something complex). Please leave out other comments from code.